src/modules/eek.marko

```diff
@@ -138,6 +138,7 @@
         </tbody>
     </table>
 
```

Comment:
I realize this isn't necessarily part of this issue, but can we include some details about what EEK is in the docs. It seems to jump right into the component examples without much explanation about what it's for. 

<kbd><img width="607" alt="image" src="https://github.com/user-attachments/assets/32e14976-5543-4a0a-bedb-58ee3eb9610e"></kbd>

Reply:
Done.